### PR TITLE
Add GitHub as a sponsor

### PR DESCRIPTION
**Pull request by ekohl**
GitHub kindly upgraded our Free plan to Team, which among other things, gives us additional capacity when using GitHub Actions.

https://community.theforeman.org/t/our-github-organization-is-now-sponsored-on-the-teams-plan/25414

### CODE REVIEW

File: sponsors.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ Thank you to gandi.net, who provides domain registration and the DNS that powers
 <div class="lead text-left"><a href="https://www.conova.com"><img src="/static/images/sponsors/conova.png" height="125px"/></a></div>
 Thank you to conova, who provides servers for our continuous integration system.
 
+### GitHub
+
+<div class="lead text-left"><a href="https://github.com"><img src="/static/images/sponsors/github.png" height="125px"/></a></div>
+Thank you to GitHub, who upgrades our organization's plan.
```
Comment (melcorr) [inline sponsors.md]:
```suggestion
Thank you to GitHub, who have provided a permanent free upgrade of our organization's plan.
```

**Review by melcorr — COMMENTED** [review]
One suggestion. The continuous present makes it sound like we're getting infinite and constant upgrades.

**Comment by ekohl** [comment]
I considered it as an ongoing action so the present tense. Normally the plan has a monthly cost but now it's free. Quoting the original email:
> As an open source project, I'm happy to give Foreman a permanent free upgrade to the Teams plan

**Comment by melcorr** [comment]
> I considered it as an ongoing action so the present tense. Normally the plan has a monthly cost but now it's free. Quoting the original email:
> 
> > As an open source project, I'm happy to give Foreman a permanent free upgrade to the Teams plan

This is great news. The tense used gives the illusion of the upgrade being continuous, as in we just keep moving up levels, rather than permanent. I have changed my suggestion.

**Review by melcorr — APPROVED** [review]
LGTM thanks

**Pull request merged**